.github/workflows/gradle.yml

```diff
@@ -15,19 +12,22 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Repository
+        uses: actions/checkout@v3
         with:
           fetch-depth: 0
-      - uses: actions/setup-java@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3
         with:
           distribution: 'temurin'
-          java-version: '17'
+          java-version: 17
           cache: 'gradle'
-      - name: Grant execute permission for gradlew
-        run: chmod +x gradlew
-      - name: Build the mod
-        run: ./gradlew --no-daemon build
-      - uses: actions/upload-artifact@v2
+      - name: Build ModernFix using Gradle
+        run: |
+          chmod +x gradlew
+          ./gradlew --no-daemon build
+      - name: Upload Artifacts to GitHub
+        uses: actions/upload-artifact@v3
         with:
-          name: Package
+          name: ModernFix-Artifacts
```

Comment:
Thanks! This name has to stay the same to avoid breaking the existing nightly links.

Reply:
Doesn't this apply to newer builds only or is it regardless of it?

Reply:
The link is intended to always point to the latest build, so it would cause the old links to no longer update.

Reply:
Undoing it then.